internals/serve-with-csp.json

```diff
@@ -0,0 +1,15 @@
+{
+  "public": "./build",
+  "headers": [
+    {
+      "source": "**",
+      "headers": [
+        { "key": "Comment", "value": "Keep Content-Security-Policy synced with deployment" },
```

Comment:
Could we point to the private k8s script we use for deployment here (@tjanez)? Otherwise CSP settings here will be difficult to sync for newcomers.